R/sem.R

```diff
@@ -989,17 +1000,21 @@ checkLavaanModel <- function(model, availableVars) {
   fitin$setExpectedSize(rows = 1, cols = 2)
 
   # information criteria
-  fitms[["incrits"]] <- fitic <- createJaspTable(gettext("Information criteria"))
-  fitic$addColumnInfo(name = "index", title = "",               type = "string")
-  if (length(options[["models"]]) < 2) {
-    fitic$addColumnInfo(name = "value", title = gettext("Value"), type = "number", format = "sf:4;dp:3")
-  } else {
-    for (i in seq_along(options[["models"]])) {
-      fitic$addColumnInfo(name = paste0("value_", i), title = options[["models"]][[i]][["modelName"]], type = "number",
-                          format = "sf:4;dp:3")
+  if (!options$estimator %in% c("DWLS", "GLS", "WLS", "ULS")) {
+    fitms[["incrits"]] <- fitic <- createJaspTable(gettext("Information criteria"))
+    fitic$addColumnInfo(name = "index", title = "",               type = "string")
+    if (length(options[["models"]]) < 2) {
+      fitic$addColumnInfo(name = "value", title = gettext("Value"), type = "number", format = "sf:4;dp:3")
```

Comment:
I am not sure why this formatting was added in the first place, but it seems redundant (you can remove it for the other columns as well).
```suggestion
      fitic$addColumnInfo(name = "value", title = gettext("Value"), type = "number")
```